__tests__/app.test.js

```diff
@@ -167,6 +157,79 @@ describe("app", () => {
 
   });
 
+
+  test('Returns Error with 404 when the article id it does not exist', () => {
+    const article_id = 50;
+    return request(app)
+   .get(`/api/articles/${article_id}/commentS`)
+    .expect(404)
+    .then(({body})=>{
+    expect(body.msg).toBe("Article not found")
+
+      });
+    
+  });
+
+  test('Returns Error with 404 if the article id is invalid', () => {
+    
+    return request(app)
+   .get("/api/articles/article/commentS")
+    .expect(400)
+    .then(({body})=>{
+    expect(body.msg).toBe("Invalid Path")
+
+      });
+    
+  });
+
   
+
+  describe("POST /api/articles/:article_id/comments", () => {
```

Comment:
We don't need the array to be returned here just the created comment object (on a key of comment or similar). Arrays are for more than one item and we are only returning one item - the comment. We can make this happen by returning just the comment object from the Model to the Controller.